src/screens/SearchResultsScreen.tsx

```diff
@@ -42,8 +43,9 @@ export default function SearchResultsScreen(): JSX.Element {
         handler();
     }, [query]);
 
-    // TODO: #194 Make skeleton loading screen
-    if (loading) return <p data-testid='search-results-loader'>Loading...</p>;
+    if (loading) {
+        return <ShowCardPlaceholder count={5} />;
```

Comment:
@Thenlie Don't we want `count` to equal `movieDetails.length + tvShowDetails.length` ?

Reply:
`movieDetails` only becomes an array after the request has been made, i.e. once loading has completed. So just going with a generic number of cards gets the point across, this is how most sites (ex. YouTube) do it.

Matching the styling closer would definitely be a good idea. Currently the show card needs quite a bit of work so I just went with a generic 'card' style which can be refactored later. If we want I can at least make the text next to the image to match the current card.

Reply:
@Thenlie gotcha. 
![image](https://user-images.githubusercontent.com/41771785/214114021-143209d3-4405-4eff-a0e2-a96998a1d6f7.png)
Looks like YouTube just has one full page of loading skeletons - which I think looks really good.

Reply:
Ah, I did think YouTube just did a single row. 
We can definitely increase the count. However, if we have less than a full page of results, it would also be inconsistent. Certainly for the Discover screen, we will increase the count as that should always return at least a full page. 

So as we use this we can just tweak the number as needed. If a full page looks better regardless of number of results, let's go with that.